methods/guest/src/main.rs

```diff
@@ -20,14 +20,14 @@ use common::output::Output;
 use common::serializable_g1::SerializableG1;
 use common::verify_blob::IVerifyBlob;
 use common::polynomial_form::PolynomialForm;
-use risc0_steel::{ethereum::EthEvmInput, Contract};
+use risc0_steel::{ethereum::{EthEvmInput,ETH_HOLESKY_CHAIN_SPEC}, Contract};
```

Comment:
```suggestion
use risc0_steel::{ethereum::{EthEvmInput, ETH_HOLESKY_CHAIN_SPEC}, Contract};
```
Have you ran cargo fmt?